packages/teams-js/src/public/dialog.ts

```diff
@@ -99,7 +99,9 @@ export namespace dialog {
    * Submit the dialog module and close the dialog
    *
    * @param result - The result to be sent to the bot or the app. Typically a JSON object or a serialized version of it
-   * @param appIds - Helps to validate that the call originates from the same appId as the one that invoked the task module
+   * @param appIds - Helps to validate that the call originates from the same appId as the one that invoked the task module.
```

Comment:
I think that this isn't *quite* accurate. I believe these are used by Teams or other hosts to confirm that the value of result is being sent back to an application with id matching one of the appids here.

Reply:
does this look good now?